west.yml

```diff
@@ -104,7 +104,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 59179ff80cee01c4feb8b747e0ff9caecb55c92d
+      revision: d74c71a0362baa906031c24275a6bab110ef31fc
```

Comment:
More convenient is following syntax. If you use it, you don't need to update this line each time dependent PR changes:
```
revision: pull/524/head
```

When dependent PR goes into master, you can remove this change.